### DISCUSSION TITLE
Energy scan during initialize

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1304] [Q&A] [answered]
During initialize() it seems that   await self.energy_scan( channels=t.Channels.ALL_CHANNELS, duration_exp=4, count=1 ) is performed. 

What is the purpose ? What is the benefit if the network is already formed ?

**Comment by TheJulianJES** [comment, marked as answer]
zigpy prints a warning message when the channel utilization of the currently used channel is high: https://github.com/zigpy/zigpy/blob/056d91401465a2d2e3b10f4d0d246f471d0d0fc8/zigpy/application.py#L188-L199

It can be disabled by setting the `startup_energy_scan` config option to `False`.
